### PR TITLE
CLOUDP-238726: AKO: Add Release 2.2.0

### PR DESCRIPTION
**Pull request by s-urbaniak**
## Proposed changes

This adds the AKO 2.1.1 release changes.

Closes #[issue number]

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Comment by s-urbaniak** [comment]
**Note** DON'T MERGE until AKO 2.1.1 release is fully done.

File: internal/kubernetes/operator/features/crds.go [hunk 1]
```diff
@@ -120,7 +130,7 @@ func CRDCompatibleVersion(operatorVersion string) (string, error) {
 	return semver.New(
 		operatorVersionSem.Major(),
 		operatorVersionSem.Minor(),
-		0,
+		operatorVersionSem.Patch(),
```
Comment (s-urbaniak) [inline internal/kubernetes/operator/features/crds.go:133]: I am not happy about this @helderjs @igor-karpukhin: I think we should declare this a 2.2 release and not 2.1.1.

**Comment by s-urbaniak** [comment]
Scheduling Kubernetes tests in evergreen first

**Comment by s-urbaniak** [comment]
all green, @gssbzn PTAL

File: internal/kubernetes/operator/features/patcher.go [hunk 2]
```diff
@@ -45,32 +45,30 @@ func NopPatcher() Patcher {
...
+		if _, ok := knownFrequencyTypes[frequencyType]; ok {
+			prunedItems = append(prunedItems, policy.Spec.Items[i])
 		}
-		policy.Spec.Items = prunedItems
+	}
+	policy.Spec.Items = prunedItems
 
-		return nil
-	})
+	return nil
```
Comment (josvazg) [inline internal/kubernetes/operator/features/patcher.go:73]: Only format changes right?

**Review by josvazg — APPROVED** [review]

File: internal/kubernetes/operator/features/patcher.go (hunk 2 above)
Comment (s-urbaniak) [reply internal/kubernetes/operator/features/patcher.go:73]: correct, this unindents the function for better readability.

**Pull request merged**